rclpy/test/listener.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ament_index_python
+import argparse
+import functools
+import importlib
+import os
+import sys
+
+# this is needed to allow rclpy to be imported from the build folder
+sys.path.insert(0, os.getcwd())
+
+
+def listener_cb(msg, message_name, received_messages):
+    print('received: %r' % msg)
+    received_messages.append(msg)
+
+
+def listener(message_pkg, message_name, rmw_implementation, number_of_cycles):
```

Comment:
Yeah sorry, this shouldn't be here and only in the following PR #29 which will actually use it